templates/azure-storage-blob/azure-storage-blob-template.variables.tf.json

```diff
@@ -7,6 +7,10 @@
     "storage_reader_users": {
       "type": "list(any)",
       "default": []
+    },
+    "enable_defender": {
+      "type": "number",
+      "default": 0
```

Comment:
would this be better as a boolean rather than an integer ?

it helps convey the intent of what being proposed